app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -70,5 +72,19 @@ def rss_notification_blueprint
     def notification_already_exists?(item)
       !!Notification.unscoped.find_by_message(item.summary)
     end
+
+    def load_rss_feed
+      uri = URI.parse(@url)
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+      request = Net::HTTP::Get.new(uri.request_uri)
+      request.initialize_http_header({"User-Agent" => rss_user_agent})
+      http.request(request).body
+    end
+
+    def rss_user_agent
+      "Foreman #{SETTINGS[:version]} RSS notifications"
```

Comment:
https://tools.ietf.org/html/rfc7231#section-5.5.3 suggests the common way is to separate the product and version with a slash. I'm also not sure about the RSS notifications part of it but if you do, convention is to put it in braces: ```Foreman/#{SETTINGS[:version]} (RSS notifications)```